A1_perception:
  ros__parameters:
    log_print: true
    # 0: unset, 10: debug, 20: info, 30: warn, 40: error
    log_level: 20
    node_params: node.yaml
